ivanrodri/react-boilerplate/readme.md

```diff
@@ -0,0 +1,76 @@
+# Nuevo react boilerplate
+
+Los miembros de **NoCountryForGeeks** hemos estado trabajando en un [proyecto base de **React**](https://github.com/NoCountryForGeeks/react-boilerplate) para que otros desarrolladores puedan aprovecharlo y usarlo para sus proyectos sin necesidad de configuración, donde podemos llegar a perder días montando la estructura inicial de nuestro proyecto para hacerlo funcionar medianamente bien,esto nos ahorrará días de proyecto lo que permitirá que nuestros clientes se ahorren un pico.
+En este proyecto base, nos hemos enfocado mucho en la experiencia de desarrollo, tener un buen entorno configurado, ágil y donde sentirnos comodos nos ayudará a ser mas productivos.
```

Comment:
Te falta un enter para que se vea en otra linea creo.

Reply:
Te refieres aquí?

![image](https://user-images.githubusercontent.com/22966198/42776648-ef2fce8a-8937-11e8-892e-7ad8d06228c9.png)

Reply:
I thinks is between:

> se ahorren un pico. 
> En este proyecto base, nos h